ansible/library/lldp_facts.py

```diff
@@ -258,7 +258,7 @@ def main():
 
     lldp_data = dict()
 
-    for intf in lldp_rem_sys.viewkeys():
+    for intf in lldp_rem_sys.keys():
```

Comment:
for loop will iter over keys by default. keys method call is redundant.

Reply:
Done. Thanks